administrate/config/locales/administrate.pt-BR.yml

```diff
@@ -0,0 +1,24 @@
+# encoding: UTF-8
+---
+pt-BR:
+  administrate:
+    actions:
+      confirm: Você tem certeza?
+      destroy: Deletar
+      edit: Editar
+      show: Visualizar
```

Comment:
Actually, ``Visualizar`` give us a sense of "View". ``Mostrar`` give us a sense of  ``displaying``, ``exhibiting``, so it makes more sense ``Mostrar/Exibir``, I've changed mostrar to exibir. It looks better!